mlir/include/air/Dialect/AIR/AIRtoAIETransformOps.td

```diff
@@ -0,0 +1,36 @@
+//===- AIRtoAIETransformOps.td ----------------------------------*- tablegen -*-===//
+//
+// Copyright (C) 2022, Xilinx Inc. All rights reserved.
+// Copyright (C) 2022, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
I guess this has to be solved up-stream?

Reply:
not sure what you mean - these are copy-pasta from adjacent tablegen - i'm not a lawyer but is it the thing that these need to be updated annually? if so i can update the adjacent ones as well